node/src/daser.rs

```diff
@@ -26,16 +26,17 @@
 //!    - If we reach a timeout of 10 seconds and at least one of the CIDs is not received, then
 //!      block is considered sampled and rejected.
 //!    - [`Store`] is updated with the sampling result.
-//! 5. Steps 3 and 4 are repeated concurently, unless we detect that all peers have disconnected.
+//! 5. Steps 3 and 4-5-6 are repeated concurently, unless we detect that all peers have disconnected.
```

Comment:
```suggestion
//! 5. Steps 3, 4, 5 and 6 are repeated concurently, unless we detect that all peers have disconnected.
```

Reply:
The reason I did it like this is because 4-5-6 run as one thing. They are the same flow.

Reply:
After some rewording @fl0rek did it made sense to merge the into one.